charts/internal/cloud-provider-config/templates/cloud-provider-config.yaml

```diff
@@ -6,3 +6,5 @@ metadata:
 data:
   cloudprovider.conf: |
     clusterName: {{ .Values.clusterName }}
+    networking:
```

Comment:
Does this need to be `cloudControllerNetworking:`?

Reply:
The fact that we are here in the `cloudprovider.conf` should be clear we are in the cloud controller context here. What is debatable is the name `networking` altogether or if there is a better name for that.

Reply:
My question arose from the `json` struct tags.

Reply:
> Does this need to be `cloudControllerNetworking:`?
From my POV no, since here we are already scoped to `cloudprovider.conf` config which is only used in CCM - so no point to give it a full name `cloudControllerNetworking`.

Same goes to the above shoot spec - the config is already for `cloudControllerManager`

Reply:
I frankly wouldn't wrap it into `cloudControllerManager`. Since it is part of the `cloudprovider.conf` it should be clear what is what.

Reply:
In the context of the GExt this makes sense :)